elasticsearch/run.sh

```diff
@@ -216,4 +216,8 @@ verify_or_add_index_templates() {
 
 verify_or_add_index_templates &
 
+if [ -v HEAP_DUMP_LOCATION ]; then
```

Comment:
they should have equivalent behaviour in bash 4.2 and newer, but I will fix according to your suggestion

https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html